### PR TITLE
Api reference: Added actor builds and runs into apiary docs.

### PR DESCRIPTION
**Pull request by drobnikj**
- Added actor builds and runs into apiary docs
- For now, I just copy and update descriptions of endpoints from Actors group.

### CODE REVIEW

**Review by mtrunkat — APPROVED** [review]

File: docs/api_v2/api_v2_reference.apib [hunk 1]
```diff
@@ -1938,6 +1938,144 @@ In order to save new items to the dataset, send HTTP POST request with JSON payl
...
+
+# Group Actor builds
+
+The API endpoints described in this section enable you to manage Apify actor build.
+
+## Build object [/v2/actor-builds/{buildId}{?token,waitForFinish}]
+
+### Get build [GET]
+
+Gets an object that contains all the details about a specific build of an actor.
```
Comment (m-murasovs) [inline docs/api_v2/api_v2_reference.apib:2042]: Same as above: would it not be good to also move Build Collection to here?
https://docs.apify.com/api/v2#/reference/actors/build-collection

File: docs/api_v2/api_v2_reference.apib [hunk 2]
```diff
@@ -1938,6 +1938,144 @@ In order to save new items to the dataset, send HTTP POST request with JSON payl
...
+The API endpoints described in this section enable you to manage Apify actor run.
+
+## Run object [/v2/actor-runs/{runId}{?token,waitForFinish}]
+
+### Get run [GET]
+
+Gets an object that contains all the details about a specific run of an actor.
+
+By passing the optional `waitForFinish=1` parameter the API endpoint will synchronously wait for the run to finish.
+This is useful to avoid periodic polling when waiting for actor run to complete.
```
Comment (m-murasovs) [inline docs/api_v2/api_v2_reference.apib:1952]: Would it not be good to have Run Collection here as well? Just to have all things related to runs in the same place?
https://docs.apify.com/api/v2#/reference/actors/run-collection

**Review by m-murasovs — CHANGES_REQUESTED** [review]

**Comment by m-murasovs** [comment]
Without changing the original `Actors` section it might be pretty confusing for users to find two bits of information that are the same except for the difference in the URL.

If it's a bad time to move the `Run Collection` and `Build Collection` out of the `Actors` section, it would be good to at least add `[Deprecated]` tags for the old `'Get Run'/'Abort Run'/etc` endpoints, no?

File: docs/api_v2/api_v2_reference.apib (hunk 1 above)
Comment (drobnikj) [reply docs/api_v2/api_v2_reference.apib:2042]: We plan to move list od builds there as well, but for now, we moved just build object as it a prerequisite for javascript client v1.

File: docs/api_v2/api_v2_reference.apib (hunk 2 above)
Comment (drobnikj) [reply docs/api_v2/api_v2_reference.apib:1952]: Same as comment above

**Comment by drobnikj** [comment]
@m-murasovs Sorry, I forgot to add deprecation warning as we discussed.

**Review by m-murasovs — APPROVED** [review]

**Pull request merged**